src/components/buttons/EdgeButton.tsx

```diff
@@ -74,233 +81,217 @@ export function EdgeButton(props: Props) {
+        testID={testID}
+        activeOpacity={Platform.OS === 'ios' ? 0.6 : 0.7}
+      >
+        {content}
+      </EdgeTouchableOpacity>
+    </View>
   )
 }
 
 const getStyles = cacheStyles((theme: Theme) => {
-  const visibleContainerCommon: ViewStyle = {
+  const pillBase: ViewStyle = {
     borderRadius: theme.rem(theme.buttonBorderRadiusRem),
-    flexGrow: 0,
-    flexShrink: 0,
+    overflow: 'visible',
     alignItems: 'center',
-    justifyContent: 'center',
-    flexDirection: 'row'
+    justifyContent: 'center'
+  }
+
+  const pillBackground: ViewStyle = {
+    ...StyleSheet.absoluteFillObject,
+    borderRadius: theme.rem(theme.buttonBorderRadiusRem)
+  }
+
+  const labelBase: TextStyle = {
+    includeFontPadding: false
+    // Defaults will be overridden per type below
   }
 
   return {
-    // Invisible Touchable Container Styles:
-    touchContainerCommon: {
-      alignItems: 'center',
-      justifyContent: 'center'
-    },
-    touchContainerSpacing: {
-      // Combination of negative margin and positive padding to increase
-      // invisible tappable area outside of the bounds of the visible button
-      margin: -theme.rem(0.5),
-      padding: theme.rem(0.5)
-    },
-    touchContainerRow: {
-      alignSelf: 'stretch',
-      flex: 1 // Size equally against other buttons in the row
-    },
-    touchContainerColumn: {
-      alignSelf: 'stretch',
-      flexBasis: 'auto',
-      flexGrow: 0,
-      flexShrink: 0
-    },
-    touchContainerFullWidth: {
-      alignSelf: 'stretch',
-      flexGrow: 1,
-      flexShrink: 0
-    },
-    touchContainerSolo: {
-      flexBasis: 'auto',
-      flexGrow: 0,
-      flexShrink: 0
-    },
-    invisibleContent: {
-      ...visibleContainerCommon,
-      opacity: 0
-    },
-    // Visible Container Styles
-    visibleContainerCommon,
-    visibleSizeDefault: {
-      paddingHorizontal: theme.rem(1.5),
-      height: theme.rem(3)
-    },
-    visibleSizeMini: {
-      alignSelf: 'center',
-      paddingHorizontal: theme.rem(1.25),
-      height: theme.rem(2)
+    pillBase,
+    pillSolid: {
+      position: 'relative'
     },
-    visibleSizeTertiary: {
-      // Reduce the bounds of a tertiary button so it doesn't appear to be too
-      // far from other buttons
-      padding: 0,
-      height: undefined
-    },
-    visibleContainerColumn: {
-      alignSelf: 'stretch'
-    },
-    visibleContainerFullWidth: {
-      alignSelf: 'stretch',
-      marginHorizontal: theme.rem(1),
-      marginBottom: theme.rem(0.5)
-    },
-    visibleContainerRow: {
+    pillStretch: {
       alignSelf: 'stretch'
     },
-    visibleContainerSolo: {
-      alignSelf: 'center'
+    pillTertiary: {
+      backgroundColor: 'transparent',
+      position: 'relative'
     },
-    visibleContainerTertiary: {
-      alignSelf: 'center'
+    pillBackground,
+    contentRow: ((): ViewStyle => ({
+      flexDirection: 'row',
+      alignItems: 'center',
+      justifyContent: 'center'
+    }))(),
```

Comment:
What's the deal with this IIFE? If it's just for types, you can do `const contentRow: ViewStyle = {...}` above in the main body of `getStyles`, and then insert it here similar to `labelBase` a few lines down.